dev/conductor/ui/macos/Runner.xcodeproj/project.pbxproj

```diff
@@ -21,6 +21,7 @@
 /* End PBXAggregateTarget section */
 
 /* Begin PBXBuildFile section */
+		2F7AB261376CBB33ED83D572 /* Pods_Runner.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = C9CFD7551DAF1A3F49CCD1A5 /* Pods_Runner.framework */; };
```

Comment:
i'm guessing we don't need all of these diffs from the ui/macos directory, since we're no longer using the pub package?

Reply:
There are some build errors I am seeing now. They should be fixed now.